spec/models/team_performance_spec.rb

```diff
@@ -74,6 +82,13 @@ def create_all_teams
         end
       end
     end
+
+    describe ".teams_to_remind" do
+      it 'should not remind anyone' do
+        create_all_teams.each { |t| t.update(season: Season.current)}
+        expect(TeamPerformance.teams_to_remind).to match_array([])
+      end
```

Comment:
This is for the removed behaviour, so I guess it'll pass ;-) but why keep it here?